atom/browser/api/atom_api_app.cc

```diff
@@ -523,6 +736,63 @@ void App::OnCertificateManagerModelCreated(
 }
 #endif
 
+#if defined(OS_WIN)
+v8::Local<v8::Value> App::GetJumpListSettings(mate::Arguments* args) {
```

Comment:
Returning `mate::Dictionary` directly would be inconsistent with the rest of Electron's bindings code.